client/src/client.rs

```diff
@@ -1741,6 +1786,12 @@ where
 		stream
 	}
 
+	fn initial_sync_import_notification_stream(&self) -> InitialSyncBlockImportNotifications<Block> {
+		let (sink, stream) = mpsc::unbounded();
```

Comment:
We should consider using bounded channel here. If someone enables offchain workers running for initial sync, most likely we probably want to back-pressure block import.

Reply:
The API for bounded channels is different, as such it would require to have a future for capacity in the mpsc which would cause quite a bit of API change trickling up the call tree. Is that ok?